app/views/admin/project_steps/_duplicate_step_form.html.slim

```diff
@@ -0,0 +1,11 @@
+= form_tag do |f|
+
+  div.form-group.row
+    input.repeat-once type="radio" name="repeat_duration"value="repeat_once" checked="checked"
+      span.bold = t("project_step.once")
+
+  div.form-group.row
+    input.custom-repeat type="radio" name="repeat_duration" value="custom_repeat"
+      span.bold = t("project_step.repeat")
+
+  = render "admin/project_steps/repeat_options", step: step, f: f
```

Comment:
I really like the way you've been choosing to organize these partials, keeps all of them pretty easily digestible and straightforward.